op-guide/build.md

```diff
@@ -0,0 +1,145 @@
+# Build
+
+## Supported platforms
+
+The following table lists TiDB support for common architectures and operating systems. 
+
+|Architecture|Operating System|Status|
+|------------|----------------|------|
+|amd64|Linux Ubuntu (14.04+)|Stable|
+|amd64|Linux CentOS (7+)|Stable|
+|amd64|Mac OSX|Experimental|
+
+## Prerequisites
+
++ Go [1.5+](https://golang.org/doc/install)
++ Rust [nightly version](https://www.rust-lang.org/downloads.html)
++ GCC 4.8+ with static library
+
+### Install GO
+
+#### Linux
+
+```bash
+curl -L https://storage.googleapis.com/golang/go1.6.3.linux-amd64.tar.gz -o golang.tar.gz
+tar -C /usr/local -xzf golang.tar.gz
+```
+
+#### Mac OS X
+
+```bash
+curl -L https://storage.googleapis.com/golang/go1.6.3.darwin-amd64.tar.gz -o golang.tar.gz
+tar -C /usr/local -xzf golang.tar.gz
+```
+
+### Install Rust
+
+```bash
+curl -sSf https://static.rust-lang.org/rustup.sh | sh -s -- --channel=nightly
+```
+
+### Install GCC
+
+#### Linux Ubuntu
+
+```bash
+apt-get update
+apt-get install -y gcc g++
+```
+
+#### Linux CentOS
+
+```bash
+yum install -y gcc-c++ glibc-static libstdc++-static
+```
+
+#### Mac OS X
+
+```bash
+xcode-select --install
+brew update
+brew tap homebrew/versions
+brew install gcc48 --use-llvm
+```
+
+## Build TiDB project
```

Comment:
The following sections are just code snippets without any explanation. Please include only the necessary code and add explanations as the leading sentence for each section.